doc/index.rst

```diff
@@ -1,4 +1,69 @@
+===============
+ts_Electrometer
+===============
+A CSC for the Electrometer.
+Released as needed.
 
-.. todo:: add autogen docstrings
+Dependencies
+============
 
-.. todo:: Write documentation
\ No newline at end of file
+.. include:: ../requirements.txt
+
+
+Using ts_Electrometer
+=====================
+This package is compatible with the scons build system and eups package management system.
+
+.. code::
+
+    setup -kr .
+    scons
+    scons install declare
+    package-docs build
+    python bin/RunElectrometer.py 1
+
+Developing ts_Electrometer
+==========================
+Developing the ts_Electrometer package follows the TSSW developer guide.
+Code style is PEP-8 compliant with a few exceptions.
+Flake8 is a tool given by the development docker image for checking PEP-8 compliance.
+The setup.cfg file comes with the necessary exceptions already configured.
+Developing with docker is described in the TSSW Developer guide.
+`ts_salobj Documentation <https://ts-salobj.lsst.io>`_ will help you develop the CSC.
+Writing unit tests is done by using the unittest module style.
+
+There is a mock_server module.
+Its not fully fleshed out yet.
+Currently you create a `socat` virtual serial port, then start the mock server using the master port as the argument.
+Commands can then be sent to the slave port and ideally any electrometer commands would respond similar to the real thing.
+However, the only command that's really implemented is the *IDN command.
```

Comment:
The * is going to be interpreted as the start of italics. If it is part of the command name then consider those double back-ticks again:
```
 ``*IDN``
```